Makefile

```diff
@@ -268,6 +268,23 @@ bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metada
 	$(SED) -e 's/    createdAt: .*/$(shell grep -I '^    createdAt: ' bundle/manifests/oadp-operator.clusterserviceversion.yaml)/' bundle/manifests/oadp-operator.clusterserviceversion.yaml > bundle/manifests/oadp-operator.clusterserviceversion.yaml.tmp
 	mv bundle/manifests/oadp-operator.clusterserviceversion.yaml.tmp bundle/manifests/oadp-operator.clusterserviceversion.yaml
 
+.PHONY: kunflux-bundle
+kunflux-bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metadata for konflux, then validate generated files.
+	GOFLAGS="-mod=mod" $(OPERATOR_SDK) generate kustomize manifests -q
+	cd config/manager && GOFLAGS="-mod=mod" $(KUSTOMIZE) edit set image controller=$(IMG)
+	GOFLAGS="-mod=mod" $(KUSTOMIZE) build config/manifests | GOFLAGS="-mod=mod" $(OPERATOR_SDK) generate bundle $(BUNDLE_GEN_FLAGS) --output-dir konflux/bundle
+	@make nullables
+	# Copy updated bundle.Dockerfile to CI's Dockerfile.bundle
+	# TODO: update CI to use generated one
+	cp konflux/Dockerfile-bundle.oadp build/Dockerfile.bundle
+	GOFLAGS="-mod=mod" $(OPERATOR_SDK) bundle validate ./konflux/bundle
+	$(SED) -e 's/    createdAt: .*/$(shell grep -I '^    createdAt: ' konflux/bundle/manifests/oadp-operator.clusterserviceversion.yaml)/' konflux/bundle/manifests/oadp-operator.clusterserviceversion.yaml > konflux/bundle/manifests/oadp-operator.clusterserviceversion.yaml.tmp
+	mv konflux/bundle/manifests/oadp-operator.clusterserviceversion.yaml.tmp konflux/bundle/manifests/oadp-operator.clusterserviceversion.yaml
+
```

Comment:
why not parameterize bundle: to acceptdifferent output dir? you can then set env to output dir to konflux/

Reply:
this looks duplicated code to me.